frontend/packages/client/src/components/common/Toast.js

```diff
@@ -1,41 +1,67 @@
-import { useHistory } from 'react-router-dom';
 import { Svg } from '@cast/shared-components';
 
-const Toast = ({ onClose, text, footerText, footerLink }) => {
-  const history = useHistory();
+const TOAST_PROPS = {
+  loading: {
+    bgColor: 'white',
+    borderColor: 'black',
+    icon: <img src="/spinner.gif" alt="loading spinner" />,
+  },
+  success: {
+    bgColor: '#F5FBF6',
+    borderColor: '#2BA148',
+    icon: <Svg name="CheckMark" />,
+  },
+  info: {
+    bgColor: '#FEF4CD',
+    borderColor: '#C8A104',
```

Comment:
hardcoding these color styles here because theyre only used in the toast (vary slightly from other colors we use) and to avoid more `App.sass` bloat